scripts/tests/export.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -eu
+
+earthly=${earthly:=earthly}
+earthly=$(realpath $earthly)
+echo "running tests with $earthly"
+
+# prevent the self-update of earthly from running (this ensures no bogus data is printed to stdout,
+# which would mess with the secrets data being fetched)
+date +%s > /tmp/last-earthly-prerelease-check
+
+# ensure earthly login works (and print out who gets logged in)
+$earthly account login
```

Comment:
I think this is needed for shellcheck

```suggestion
"$earthly" account login
```

Reply:
it's actually not complaining about this; but it definitely breaks if I'm working out of a directory that has a space in it; i'll fix this in all of our scripts.